onadata/libs/utils/presigned_download_url.py

```diff
@@ -0,0 +1,44 @@
+import boto3
+from botocore.exceptions import ClientError
+import logging
+
+from django.conf import settings
+from django.core.files.storage import get_storage_class
+from django.http import HttpResponse, HttpResponseRedirect
+
+from wsgiref.util import FileWrapper
+
+
+def generate_media_download_url(file_path: str, expiration: int = 3600):
+    default_storage = get_storage_class()()
+    filename = file_path.split("/")[-1]
+    s3 = get_storage_class('storages.backends.s3boto3.S3Boto3Storage')()
+
+    if default_storage.__class__ != s3.__class__:
```

Comment:
What happens if the s3 packages aren't installed?

Reply:
Yes it does on travis, I wasn't sure how to handle that other than adding the requirements on Travis

Reply:
Cool, does it raise an exception or something of the sort? We could keep these lines within a `try/except` block if so; Continues on with the previous/default URL retrieval procedures if the exception is raised or something of the sort

Reply:
Could you catch the exception `ModuleNotFoundError: No module named 'storages'` raised and use the default action?

Reply:
cool I can do that